Py/DICOMSegmentationPlugin.py

```diff
@@ -143,54 +143,67 @@ def load(self,loadable):
       (success,labelNode) = slicer.util.loadLabelVolume(labelFileName, returnNode=True)
 
       # Initialize color and terminology from .info file
-      # Format of the .info file:
+      # See SEG2NRRD.cxx for how it's written.
+      # Format of the .info file (no leading spaces, modifiers are optionally written if present):

```

Comment:
I will. At the moment, working on another extension that also needs to go to Slicer asap.